bedrock/firefox/templates/firefox/family/index.html

```diff
@@ -14,7 +14,9 @@
 {% block page_title_prefix %}The Tech Talk{% endblock %}
 {% block page_title_suffix %}{% endblock %}
 
-{% block page_image %}{{ static('protocol/img/logos/firefox/browser/og.png') }}{% endblock %}
+{% block page_image %}{{ static('img/firefox/family/meta-image.jpg') }}{% endblock %}
+{% block page_og_title %}Firefox for families{% endblock %}
+{% block page_og_desc %}How to make an awkward conversation about online safety with your kid slightly less awkward.{% endblock %}
```

Comment:
Unless the OpenGraph description differs from the regular description, this could go in the `page_desc` block and `page_og_desc` will draw from that.

Reply:
Cool, will fix